OrbitCore/TracepointEventBuffer.cpp

```diff
@@ -72,3 +81,15 @@ void TracepointEventBuffer::ForEachTracepointEventOfThreadInTimeRange(
   ForEachTracepointEventInEventMapInTimeRange(min_tick, max_tick, GetTracepointsOfThread(thread_id),
                                               action);
```

Comment:
I think you should put this call in the `else` branch. Previously, an early return was used. But now there's an early return in the `else if` branch, but not in the `if` branch.

Reply:
fixed it